nexus.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Nexus
+  hosts: your_gcp_vm_group
```

Comment:
You may forgot to specify inventory group. It would better to use `server_nexus` group.